packages/ui/src/tracking/index.ts

```diff
@@ -34,6 +34,10 @@ export type GtmEventParams = {
   search_count?: number;
   percent_scrolled?: number; // 0-100 percent scrolled refers to the percentage of the page that has been viewed so far
   noninteraction: boolean; // false if an event is not tracked as a user-generated action
+  uxSessionId?: string | ''; // unique session id generated by the survey app to help correlating DB records
```

Comment:
do we need empty string type here? I think `uxSessionId?: string` should be enough?